app/services/importer/get_bamboo_fields.rb

```diff
@@ -0,0 +1,319 @@
+# frozen_string_literal: true
+
+module Importer
+  class GetBambooFields < GetApiFields
```

Comment:
Just confirming this is only included as the old version - expecting something to look different here